devbox.json

```diff
@@ -0,0 +1,40 @@
+{
+  "packages": [
+    "stdenvNoCC@latest",
+    "go@1.21.4",
+    "pre-commit@3.3.3",
+    "llvm_14@14.0.6",
+    "lld_14@14.0.6",
+    "pkg-config@0.29.2",
+    "clang_14@14.0.6",
+    "clang-tools@14.0.6",
+    "autoconf@2.71",
+    "gettext@0.21.1",
+    "automake@1.16.5",
+    "libtool@2.4.7",
+    "glibc.dev",
+    "glibc.static",
+    "llvmPackages_14.stdenv@latest",
+    "llvmPackages_14.bintools@latest",
+    "llvmPackages_14.lld@latest"
+  ],
```

Comment:
What's the process to update the packages?

Maybe we need to configure a new [regex manager](https://docs.renovatebot.com/modules/manager/regex/) with Renovate? The [Repology datasource](https://docs.renovatebot.com/modules/datasource/repology/) could be of use